content/npm-provenance-ga.md

```diff
@@ -0,0 +1,16 @@
++++
+title = "npm's Sigstore-powered provenance goes GA"
+author = "The Sigstore Technical Steering Committee"
+date = "2023-10-03"
+draft = false
+type = "post"
+tags = ["sigstore","npm","security","node"]
++++
+
+Last week saw the [GA release of npm CLI’s native Sigstore functionality](https://github.blog/changelog/2023-09-26-npm-provenance-general-availability/), a project [over a year in the making](https://www.wired.com/story/github-code-signing-sigstore/). This is a tremendous milestone for the adoption of Sigstore in open source projects and represents huge progress in effecting a cultural shift toward expecting provenance to exist for software components. 
+
+During the [public beta period](https://blog.sigstore.dev/npm-public-beta/) (which lasted from April 2023 until September 26), over 3,800 projects have adopted build provenance (including 134 high-impact projects), resulting in over 500 million total downloads of provenance-enabled package versions to-date. 
+
+npm is the world’s largest package registry, handling tens of billions of downloads per month and around 40k publishing events each day. The decision to place [SLSA-compliant](https://slsa.dev/spec/v1.0/provenance) provenance-generation functionality into the CLI directly (as opposed to locating it in a CI job or external tool) represents bold leadership in the community of package management tooling. In essence, npm is saying that package maintainers ought to have access to first-class tooling for producing authentic information about the source code and build instructions that produced their package. The effort to produce that functionality in npm resulted in the creation of two powerful new JavaScript libraries: the [sigstore-js](https://github.com/sigstore/sigstore-js) package which produces and verifies Sigstore signatures, and [tuf-js](https://github.com/theupdateframework/tuf-js), which enables the npm CLI to use Sigstore’s TUF trust root to ensure secure communications with the public good instance. Both of these libraries have been donated to their respective organizations, ensuring that the work done by the npm team can have positive network effects in the future.
```

Comment:
```suggestion
npm is the world’s largest package registry, handling tens of billions of downloads per month and around 40k publishing events each day. The decision to place [SLSA-compliant](https://slsa.dev/spec/v1.0/provenance) provenance-generation functionality into the CLI directly (as opposed to locating it in a CI job or external tool) represents bold leadership in the community of package management tooling. In essence, npm is saying that package maintainers ought to have access to first-class tooling for producing authentic information about the source code and build instructions that produced their package. The effort to produce that functionality in npm resulted in the creation of two powerful new JavaScript libraries: the [sigstore-js](https://github.com/sigstore/sigstore-js) package which produces and verifies Sigstore signatures over artifacts and attestations, and [tuf-js](https://github.com/theupdateframework/tuf-js), which enables the npm CLI to use Sigstore’s TUF trust root to ensure secure communications with the public good instance. Both of these libraries have been donated to their respective organizations, ensuring that the work done by the npm team can have positive network effects in the future.
```

Reply:
That's hard to see the difference: I added "over artifacts and attestations" after "verifies Sigstore signatures" to emphasize that the library supports attestations too (since that's still a WIP for some sigstore libraries)